### PR TITLE
added "on" property 

### PR DESCRIPTION
**Pull request by jpnbastos**
…CameraUsage

related with Issue #25

### CODE REVIEW

File: Camera/schemaDTDL.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
             "displayName": "cameraUsage",
```
Comment (albertoabellagarcia) [inline Camera/schemaDTDL.json:44]: this file is generated automatically. You do not need it on the PR (we try you to work the less the better)

File: Camera/examples/example-geojsonfeature.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
       ]
```
Comment (albertoabellagarcia) [inline Camera/examples/example-geojsonfeature.json:27]: this file is generated automatically. You do not need it on the PR (we try you to work the less the better)

File: Camera/schema.json [hunk 3]
```diff
@@ -43,7 +43,7 @@
             "DOME",
             "DAY/NIGHT",
             "C-MOUNT",
-            "BULLET"
+            "BULLET"
```
Comment (albertoabellagarcia) [inline Camera/schema.json:46]: Need TRAFFIC option in the enumeration

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
Fix these tiny changes and it will be merged.
And include the ADOPTERS.yaml where this example is running.
if you want to add your name to the contributors.yaml file please do it so.
thanks

File: Camera/schemaDTDL.json (hunk 1 above)
Comment (jpnbastos) [reply Camera/schemaDTDL.json]: done

File: Camera/schema.json (hunk 3 above)
Comment (jpnbastos) [reply Camera/schema.json:46]: hmm initially I considered extending this, but this is not our goal. We just want a description of its usage to be for Traffic, so we think its best if it goes with the enum in cameraUsage. But it looks as if this is a free enumerator.

**Review by albertoabellagarcia — APPROVED** [review]

File: Camera/schema.json (hunk 3 above)
Comment (albertoabellagarcia) [reply Camera/schema.json:46]: ok, then please remove it from the description

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
See last comment regarding traffic

File: Camera/schema.json (hunk 3 above)
Comment (albertoabellagarcia) [reply Camera/schema.json:46]: I am not seeing traffic removed. Am I wrong?

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
traffic is it removed?

**Comment by jpnbastos** [comment]
Sorry for taking so long, did not notice the comment

**Review by albertoabellagarcia — APPROVED** [review]
ok

**Pull request merged**